apps/global/src/markdown-pages/speakers/hanna-swatz.pt.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
This file name needs renamed. It has a typo

Reply:
Pulled requested files out of .gitignore.
Corrected typo on file name.